src/app/api/file-upload/[slug]/route.ts

```diff
@@ -35,8 +35,8 @@ const fileUploadApiHandler_ = async function POST(
 
 export { fileUploadApiHandler_ as POST };
 
-export const config = {
-  api: {
-    bodyParser: false,
-  },
-};
\ No newline at end of file
+// export const config = {
```

Comment:
@brianmuks this was failing the build, check docs, it seems new nextjs version don't require this but please do check if it is still needed.